float32 base_pitch
float32 diff_1_pitch
float32 diff_2_pitch
float32 diff_1_roll
float32 diff_2_roll
float32 end_effector

